### ISSUE TITLE
Moving on from Katacoda

### ISSUE THREAD

**Issue by Debanitrkl** [kubernetes-sigs/contributor-katacoda#14]
As the [katacoda.com has shut down](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html) from 15th of June, and our [scenario](https://katacoda.com/k8scontributors) is no longer available we have to look up for other alternatives outside to keep this contributor-katacoda running. 
Here are few of the alternatives to look after:
1. https://killercoda.com/, also some discussions regarding this has been mentioned here: https://github.com/kubernetes/website/issues/33918#issuecomment-1144786114
2. https://instruqt.com/, this suggestion came as a discussion here: https://github.com/kubernetes/website/issues/33918#issuecomment-1136372030

**Comment by chris-short** [comment]
Blasted things

**Comment by MadhavJivrajani** [comment]
Interesting 
According to the source of truth: https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml, the contributor-comms label is only enabled on a select few repos. We'll need to add an entry for this repo in labels.yaml if we want to use that label here.

**Comment by Akshit42-hue** [comment]
Hello Folks, I have recently migrated Katacoda(Thanos) to Killercoda(Thanos) if you want any help I am happy to help you out 🙂

**Comment by Debanitrkl** [comment]
Hi @Akshit42-hue thanks for your interest, could you please try and make a POC for the scenario-setup-linux katacoda in Killercoda so that it could give us an idea on how things work on Killercoda because at the moment we are also considering using GitHub codespace.

**Comment by Akshit42-hue** [comment]
Yes @Debanitrkl, will be creating a Killercoda scenario and sharing the link with you ASAP

**Comment by craiglpeters** [comment]
I'd be thrilled to help get this going with GitHub Codespaces - I am a Codespaces product manager. Are the scenarios that were on https://katacoda.com/k8scontributors archived anywhere?

**Comment by Debanitrkl** [comment]
> I'd be thrilled to help get this going with GitHub Codespaces - I am a Codespaces product manager. Are the scenarios that were on https://katacoda.com/k8scontributors archived anywhere?

No the oriely platform isn't available now so it couldn't be archived.

**Comment by Akshit42-hue** [comment]
Hey @Debanitrkl, here is the [link](https://killercoda.com/akshit/scenario/Scenario-setup-linux) for Killercoda. Sorry for the delay
